packages/blade/src/components/Typography/Text/__tests__/Text.web.test.tsx

```diff
@@ -70,7 +86,7 @@ describe('<Text />', () => {
     expect(() =>
       renderWithTheme(
         // @ts-expect-error testing failure case when size='small' is passed with variant='caption'
-        <Text type="normal" variant="caption" truncateAfterLines={3} size="small">
+        <Text color="normal" variant="caption" truncateAfterLines={3} size="small">
```

Comment:
Color normal is not valid, right? We're testing for incorrect size & not incorrect color here